src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4205,7 +4205,7 @@ .FORWARDHELPCATEGORY Cmdlet
 param(
     [Parameter(ParameterSetName='nameSet', Position=0, ValueFromPipelineByPropertyName=$true)]
     [Parameter(ParameterSetName='pathSet', Mandatory=$true, Position=0, ValueFromPipelineByPropertyName=$true)]
-    [System.String[]]
+    [System.string[]]
```

Comment:
This one shouldn't be changed, since we are using the full type name.

Reply:
```suggestion
    [System.String[]]
```

Reply:
Fixed.